docs/book/how-to/setting-up-a-project-repository/access-management.md

```diff
@@ -0,0 +1,80 @@
+---
+description: A guide on managing user roles and responsibilities in ZenML.
+---
+
+# Access Management and Roles in ZenML
+
+Effective access management is crucial for maintaining security and efficiency in your ZenML projects. This guide will help you understand the different roles within a ZenML server and how to manage access for your team members.
+
+## Typical Roles in an ML Project
+
+In an ML project, you will typically have the following roles:
+
+- Data Scientists: Primarily work on developing and running pipelines.
+- MLOps Platform Engineers: Manage the infrastructure and stack components.
+- Project Owners: Oversee the entire ZenML deployment and manage user access.
+
+The above is an estimation of roles that you might have in your team. In your case, the names might be different or there might be more roles, but you can relate the responbilities we discuss in this document to your own project loosely.
+
+{% hint style="info" %}
+You can create [Roles in ZenML Pro](../../getting-started/zenml-pro/roles.md) with a given set of permissions and assign them to either Users or Teams that represent your real-world team structure. Sign up for a free trial to try it yourself: https://cloud.zenml.io/
+{% endhint %}
+
+## Service Connectors: Gateways to External Services
+
+Service connectors are how different cloud services are integrated with ZenML. They are used to abstract away the credentials and other configurations needed to access these services.
+
+Ideally, you would want that only the MLOps Platform Engineers have access for creating and managing connectors. This is because they are closest to your infrastructure and can make informed decisions about what authentication mechanisms to use and more.
+
+Other team members can use connectors to create stack components that talk to the external services but should not have to worry about setting them and shouldn't have access to the credentials used to configure them.
+
+Learn more about the best practices in managing credentials and recommended roles in our [Managing Stacks and Components guide](../stack-deployment/README.md).
+
+
+## ZenML Server Upgrades
```

Comment:
why is this in access management? seems off

Reply:
Yep def belongs somewhere else

Reply:
this is not a section about zenml upgrades. it's a section on what role should handle zenml upgrades. this whole page to me was about user roles and responsibilities in ZenML. access management is one part of it. 

```
---
description: A guide on managing user roles and responsibilities in ZenML.
---
```

from the guiding docs that you wrote, you also wanted this page to be about this. we already have access management specifically in other places. this is more about managing user responsibilities. same reason why pipeline management is here too.
https://www.notion.so/zenml/Best-practices-for-setting-up-a-ZenML-project-11ff8dff2538805f9646fc6eb5592a3a?pvs=4#11ff8dff253880fbb1fad3dc85f003a5